mail/urls.py

```diff
@@ -16,4 +16,9 @@
     url(r'^api/v0/mail/course/(?P<course_id>[\d]+)/$', CourseTeamMailView.as_view(), name='course_team_mail_api'),
     url(r'^api/v0/mail/learner/(?P<student_id>[\d]+)/$', LearnerMailView.as_view(), name='learner_mail_api'),
     url(r'^api/v0/mail/automatic_email/$', AutomaticEmailView.as_view(), name='automatic_email_api'),
+    url(
```

Comment:
btw the approach I was using before seemed to work alright with `reverse`, I could pass `kwargs={'email_id': id}` to get a `PATCH` url or just leave it off otherwise.